### PR TITLE
Support pebble-ready hooks in debug-hooks.

### PR DESCRIPTION
**Pull request by hpidcock**
Pulls the charm meta from the client to get the correct list of hooks.

## QA steps

juju debug-hooks myapp/0 mycontainer-pebble-ready

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1931570

### CODE REVIEW

File: cmd/juju/commands/debughooks.go [hunk 1]
```diff
@@ -174,36 +185,23 @@ func (c *debugHooksCommand) validateHooksOrActions() error {
...
-	return validHooks, nil
+func (c *debugHooksCommand) getValidHooks(ch charm.Charm) (set.Strings, error) {
+	hooks := set.Strings(ch.Meta().Hooks())
+	return hooks.Union(set.NewStrings(
+		"juju-info-relation-broken",
+		"juju-info-relation-changed",
+		"juju-info-relation-created",
+		"juju-info-relation-departed",
+		"juju-info-relation-joined",
+	)), nil
```
Comment (wallyworld) [inline cmd/juju/commands/debughooks.go:204]: We should still construct the "juju-info" hooks by iterating over hooks.RelationHooks()

```
for _, hook := range hooks.RelationHooks() {
	hook := fmt.Sprintf("juju-info-%s", relation, hook)
	hooks.Add(hook)
}
```

**Review by wallyworld — APPROVED** [review]
Add a test? Or no changes to existing test?

**Comment by hpidcock** [comment]
[Debug-hooks fix for pebble-ready](https://trello.com/c/4Phm22kn/1640-debug-hooks-fix-for-pebble-ready)

**Comment by hpidcock** [comment]
> Add a test? Or no changes to existing test?

Current tests have enough coverage.

**Comment by hpidcock** [comment]
$$merge$$

**Pull request merged**